recipes/libunrar/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "libunrar" %}
+{% set version = "6.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.rarlab.com/rar/unrarsrc-{{ version }}.tar.gz
+  sha256: 120936e41f826cd63d77a580aeea64c1b79fd3e2434f58ce8184e783b51e5b01
+  patches:
+    - 0001-REMOVE-HARDCODED-MAKEFILE-ENV.patch  # [unix]
+    - 0002-REMOVE-INSTALL-OPTION.patch  # [osx]
+    - 0003-CHANGE-DYLIB-EXTENSION.patch  # [osx]
+
+build:
+  run_exports:
+    - {{ pin_subpackage('libunrar', exact=True) }}
```

```suggestion
    - {{ pin_subpackage('libunrar', max_pin='x.x.x') }}
```
Exact means including the build string. You said the ABI changes with every patch version.

---

I see. Thanks!